library/Deprecated.php

```diff
@@ -0,0 +1,203 @@
+	class Requests_Exception_HTTP_416 extends WpOrg\Requests\Exception\Http\Status416 {}
+}
+if (class_exists('Requests_Exception_HTTP_417') === false && class_exists('WpOrg\Requests\Exception\Http\Status417') === true) {
+	class Requests_Exception_HTTP_417 extends WpOrg\Requests\Exception\Http\Status417 {}
+}
+if (class_exists('Requests_Exception_HTTP_418') === false && class_exists('WpOrg\Requests\Exception\Http\Status418') === true) {
+	class Requests_Exception_HTTP_418 extends WpOrg\Requests\Exception\Http\Status418 {}
+}
+if (class_exists('Requests_Exception_HTTP_428') === false && class_exists('WpOrg\Requests\Exception\Http\Status428') === true) {
+	class Requests_Exception_HTTP_428 extends WpOrg\Requests\Exception\Http\Status428 {}
+}
+if (class_exists('Requests_Exception_HTTP_429') === false && class_exists('WpOrg\Requests\Exception\Http\Status429') === true) {
+	class Requests_Exception_HTTP_429 extends WpOrg\Requests\Exception\Http\Status429 {}
+}
+if (class_exists('Requests_Exception_HTTP_431') === false && class_exists('WpOrg\Requests\Exception\Http\Status431') === true) {
+	class Requests_Exception_HTTP_431 extends WpOrg\Requests\Exception\Http\Status431 {}
+}
+if (class_exists('Requests_Exception_HTTP_500') === false && class_exists('WpOrg\Requests\Exception\Http\Status500') === true) {
+	class Requests_Exception_HTTP_500 extends WpOrg\Requests\Exception\Http\Status500 {}
+}
+if (class_exists('Requests_Exception_HTTP_501') === false && class_exists('WpOrg\Requests\Exception\Http\Status501') === true) {
+	class Requests_Exception_HTTP_501 extends WpOrg\Requests\Exception\Http\Status501 {}
+}
+if (class_exists('Requests_Exception_HTTP_502') === false && class_exists('WpOrg\Requests\Exception\Http\Status502') === true) {
+	class Requests_Exception_HTTP_502 extends WpOrg\Requests\Exception\Http\Status502 {}
+}
+if (class_exists('Requests_Exception_HTTP_503') === false && class_exists('WpOrg\Requests\Exception\Http\Status503') === true) {
+	class Requests_Exception_HTTP_503 extends WpOrg\Requests\Exception\Http\Status503 {}
+}
+if (class_exists('Requests_Exception_HTTP_504') === false && class_exists('WpOrg\Requests\Exception\Http\Status504') === true) {
+	class Requests_Exception_HTTP_504 extends WpOrg\Requests\Exception\Http\Status504 {}
+}
+if (class_exists('Requests_Exception_HTTP_505') === false && class_exists('WpOrg\Requests\Exception\Http\Status505') === true) {
+	class Requests_Exception_HTTP_505 extends WpOrg\Requests\Exception\Http\Status505 {}
+}
+if (class_exists('Requests_Exception_HTTP_511') === false && class_exists('WpOrg\Requests\Exception\Http\Status511') === true) {
+	class Requests_Exception_HTTP_511 extends WpOrg\Requests\Exception\Http\Status511 {}
+}
+if (class_exists('Requests_Exception_HTTP_Unknown') === false && class_exists('WpOrg\Requests\Exception\Http\StatusUnknown') === true) {
+	class Requests_Exception_HTTP_Unknown extends WpOrg\Requests\Exception\Http\StatusUnknown {}
+}
```

Comment:
There seem to be two exception classes missing:
- `Exception\InvalidArgument`
- `Exception\Transport`

Reply:
1. `Exception\InvalidArgument` is a new class in Requests 2.0.0, so doesn't need to be in the BC-layer.
2. It's included in line 99.